### PR TITLE
chore(ci): Switch to Firefox stable now that 51 is out

### PR DESCRIPTION
**Pull request by heathkit**

### CODE REVIEW

File: spec/ciFullConf.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ exports.config = {
     'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
     'build': process.env.TRAVIS_BUILD_NUMBER,
     'name': 'Protractor suite tests',
-    'version': '47',
```
Comment (juliemr) [inline spec/ciFullConf.js]: Shouldn't this be 51 now?

Comment (mgiambalvo) [inline spec/ciFullConf.js]: This is still a work in progress. Saucelabs doesn't have 51 yet, and the beta appears to be broken now. I'll check back tomorrow.

Comment (aperdomob) [inline spec/ciFullConf.js]: what happen with this CR @juliemr @mgiambalvo

Comment (mgiambalvo) [inline spec/ciFullConf.js]: Saucelabs doesn't have FF 51 deployed yet, and last I checked we had issues with 50. I'll give it another look soon.

**Comment by heathkit** [comment]
haha, firefox. Oh well, let's try this again later.

**Pull request closed without merging**